src/security/auth.py

```
@@ -0,0 +1,48 @@
+from fastapi import Depends, HTTPException, status
+from fastapi.security import HTTPBearer, HTTPAuthorizationCredentials
+from jose import JWTError, jwt
+from passlib.context import CryptContext
+from datetime import datetime, timedelta
+from typing import Optional
+import os
+
+# Security settings
+SECRET_KEY = os.getenv("JWT_SECRET_KEY")
+if not SECRET_KEY:
+    raise RuntimeError("JWT secret missing; set JWT_SECRET_KEY env var")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)